src/main/java/org/grouplens/grapht/annotation/AnnotationBuilder.java

```diff
@@ -326,7 +326,31 @@ public AnnotationBuilder(Class<T> annotType) {
     public <A extends Annotation> AnnotationBuilder<T> set(String name, A[] value) {
         return set(name, value, value.getClass());
     }
-    
+
+    /**
+     * As {@link #set(String, Enum)} but assigns an Annotation instance to
```

Comment:
Should still be '#set(String, boolean)'. Also, should say 'assigns an enum value'.